package.json

```diff
@@ -94,10 +95,10 @@
         "!(packages/**/node_modules/)*.ts": [
             "prettier --write"
         ]
-      },
-      "husky": {
+    },
+    "husky": {
         "hooks": {
-          "pre-commit": "lint-staged"
+            "pre-commit": "lint-staged"
```

Comment:
why are we adding husky with this PR?

Reply:
I think this is just indentation fixing from prettier, nothing new added.